bindings/python/spatial/expose-explog.cpp

```diff
@@ -50,7 +50,7 @@ namespace pinocchio
               "Log: SE3 -> se3. Pseudo-inverse of exp from SE3"
               " -> { v,w in se3, ||w|| < 2pi }.");
       
-      bp::def("log6",&log6<Eigen::Matrix4d>,
+      bp::def("log6",&log6<Eigen::MatrixXd>,
```

Comment:
I don't think that Matrix4d is exposed. But it should be.

Reply:
It seems to be exposed